tidb-binlog/pump_client/pump.go

```diff
@@ -26,6 +26,8 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
+var localPump string = "localPump"
```

Comment:
add some annotations for it, it's a specify pump node ID used to write local pump through unix socket connection